### ISSUE TITLE
RFE/C: Mobile Interface

### ISSUE THREAD

**Issue by TPS** [astralapp/astral#220]
Per @syropian @ https://github.com/vfeskov/gitpunch/issues/51#issuecomment-419970014,
> @TPS You can't sign in on mobile because the app is not optimized for mobile.

This request is to do so, on some level.

**Comment by TPS** [comment]
I suppose most of us would assume what I did (from the screenshots): it needs to be hosted someplace, & so is essentially a developer (not end-user) tool.  I'd no idea you were hosting it @ all!

**Comment by syropian** [comment]
Yep! The option to self-host is there, but the hosted service is by far the most popular option that people lean towards.

**Comment by TPS** [comment]
If 1 knows it exists!  How 'bout a note toward the top of the site someplace:
>This site is currently desktop-only.  Visit from there, &, if you wish to chime in re: a mobile interface, chime in @ `https://github.com/astralapp/astral/issues/220`.

**Comment by syropian** [comment]
@TPS Hm, to me it seems obvious people know about the hosted option, simply by the amount of activity the app gets. There's links to the GitHub, and Twitter on the marketing site and app if people want to file issues regarding mobile. It just isn't something that's come up.
